build_tools/bazel/iree_check_test.bzl

```diff
@@ -50,6 +56,8 @@ def iree_check_test(
             "-mlir-print-op-on-diagnostic=false",
             "-iree-hal-target-backends=%s" % target_backend,
         ] + compiler_flags,
```

Comment:
Yeah I think here if you moved `compiler_flags` before the other flags, then this would work without the extra tool